runtime/interpreter/interpreter.go

```diff
@@ -292,15 +292,17 @@ var _ ast.DeclarationVisitor[StatementResult] = &Interpreter{}
 var _ ast.StatementVisitor[StatementResult] = &Interpreter{}
 var _ ast.ExpressionVisitor[Value] = &Interpreter{}
 
-// BaseActivation is the activation which contains all base declarations.
```

Comment:
This declaration should stay as-is, we shouldn't ever need to recreate the whole base activation

Reply:
ah, was just a cautionary clone since I append a binding to `log` later on, guess it doesn't mutate the base activation.

Reply:
Yeah, the parent base activation isn't modified, the activation is basically a linked list.